columnflow/production/cms/seeds.py

```diff
@@ -18,8 +18,8 @@
 @producer(
     uses={
         # global columns for event seed
-        "run", "luminosityBlock", "event", "nGenJet", "nGenPart", "nJet", "nPhoton", "nMuon",
-        "nElectron", "nTau", "nSV",
+        "run", "luminosityBlock", "event", "nGenJet", "nGenPart", "nPhoton",
+        "nSV",
```

Comment:
Is there a reason, why nGetJet, nGenPart and nPhoton are still listed here in the `uses`?

Reply:
Yes, this has to do with how the `from_root` NanoAOD loader works in `coffea`. IIUC, whenever a jagged column like `Jet_pt` is passed to the column filter, the `nJet` column is also required and is read out to work out the length of the nested lists. However, the `nJet` column is then removed by `coffea`, so the check introduced by this PR will fail if `nJet` is present under `uses`. (I assume `nJet` is still defined as a behavior, so calling `events.nJet` would still work and return the equivalent of `ak.num('Jet')`, but `nJet` is no longer an actual column in the event array). In contrast, if no columns are passed that begin with `Jet_`, then `nJet` is read as a normal column and is found by the checker.

So, because this producer reads the `Jet`, `Muon`, `Electron`, `Tau` collections explicitly, we need to remove the corresponding `n*` columns or the column presence check will fail. For `GenJet` or `GenPart` or `Photon`, including the `n*` columns is is not problematic, since the corresponding collections are not read.

Reply:
OK, thanks for clarifying